.gitignore

```diff
@@ -2,3 +2,4 @@
 vendor/
 composer.lock
 error.log
+.idea
```

Comment:
Please drop this from the commit; we're not putting vendor-specific files in here. 

You can globally ignore the `.idea` directory from git:
https://stackoverflow.com/a/7335487